### PR TITLE
make names consistent

### PR DESCRIPTION
**Pull request by wrex**
There are a lot of changes to review, but they were almost entirely search/replace of very unique patterns (the custom property names). Everything still seems to work properly.

I tried to minimize changes to the main stylesheet to keep the commit as easy to review as possible. I did try to clean up the comment indentation, though. I think as long as we only go FROM github TO openstyles and never the other direction, there shouldn't be extraneous noise in commit messages from here on out.

Changes include:

- Change property names per issue #106. 
- eliminate unused -extra-clr in script and override examples, 
- add user stylesheet header to override files,
- group override variables into categories (just comments)

The last is consistent with my (still WIP) themer app: https://wked.rexs.tools

### CODE REVIEW

**Comment by wrex** [comment]
Forgot to mention:

I'm comfortable releasing Elementary Dark 2.0.0 with or without this PR (but I'd prefer with). 

Once it's released, I'll create a separate post announcing my themer app. I've decided to hold off on releasing a companion userscript to auto-install WKED and any custom theme. It's enough for the time being to just copy the overrides to the users clipboard for them to paste into stylus. User scripts are kind of a pain to maintain, so I'd like to get the app itself fully buttoned down before adding the live-update feature and companion userscript.

File: README.md
Comment (Sepitus-exe) [inline README.md:132]: `text-clr` no longer exist
```suggestion
it needs dark colored text to remain legible. Everything else uses `text`.
```

File: README.md
Comment (Sepitus-exe) [inline README.md:139]: `inverted-surface` no longer exist
```suggestion
  `surface-inv`. Please ensure text remains legible over all your color
```

File: README.md
Comment (Sepitus-exe) [inline README.md:143]: `inverted-surface` still keeps on non-existing
```suggestion
  `surface-inv`.
```

File: README.md
Comment (Sepitus-exe) [inline README.md:150]: The example on line 214 uses no longer supported props `--ED-surface-5` and  `--ED-text-color`.

I would amend the comment onto it, but since it wasn't modified within the scope of this branch, git wont let me 😅

File: WKElementaryDark.css
Comment (Sepitus-exe) [inline WKElementaryDark.css:11]: The indents I was worried showed up after all. I don't mind them. My linux machine was also unhappy with the indentless state of the file.

File: user-overrides/All-variables.css
Comment (Sepitus-exe) [inline user-overrides/All-variables.css:12]: Just a thought about publishing.

Would it be wise to publish the override template as we plan on publishing the rest of the files. Any future changes to the published file will most likely wipe any user changes made to it and since stylus tends to update automatically by default, it could lead to bunch of issues down the line.

Subsequently, if we do not plan on publishing the override template, there is no real need for metadata.

File: userscript-styles/WKED-self-study.css
Comment (Sepitus-exe) [inline userscript-styles/WKED-self-study.css:35]: `--ED-surface-5` no longer exists
```suggestion
    background-color: var(--ED-surface-inv, #bababa);
```

File: userscript-styles/WKED-self-study.css
Comment (Sepitus-exe) [inline userscript-styles/WKED-self-study.css:56]: `--ED-text-color` no longer exists
```suggestion
    color: var(--ED-text, #eeeeee);
```

File: userscript-styles/WKED-self-study.css
Comment (Sepitus-exe) [inline userscript-styles/WKED-self-study.css:81]: `--ED-text-color` insists on not existing
```suggestion
    color: var(--ED-text, #eeeeee);
```

**Review by Sepitus-exe — COMMENTED** [review]
Few things are a bit off, but the thing works non the less, I love how robust the project turned out to be.

I still have to do a review of the main file before merging, which I was unable to complete due to time restrains. I should be able to do so in about 4-5ish hours.

File: user-overrides/All-variables.css
Comment (wrex) [reply user-overrides/All-variables.css:12]: Hmm. I see your point. Definitely don't want users just editing anything that's published to userstyles.world (since those will be autoupdated). The instructions do say to *copy* the files, not edit in place, but I think you're probably right that it's better not to publish these to userstyles.world.

I'll make this clearer in the README and in the comments to the override styles (removing the UserStyle headers).

**Comment by wrex** [comment]
This is so weird, I did global search and replace across all files in git. It appears that vscode might ignore text in comments when doing global S&R (I hate software that tries so hard to be helpful that it gets in the way). I'm not sure how self-study quiz was missed -- there may have been another merge or something that ate the changes.

Anyway, another commit coming shortly to this branch.

File: WKElementaryDark.css
Comment (wrex) [reply WKElementaryDark.css:11]: Can you provide examples of exactly what you're seeing? I'm unclear what the problem is. I use two-space indents and prettier with vscode. The file should render identically on osx or linux. I suspect that one of our platforms is inserting literal tab characters rather than spaces, and we need to NOT do that.

Comment (wrex) [reply WKElementaryDark.css:11]: Never mind. I see you meant that my one of our editors indented everything with spaces in the `@moz-document` block. I just checked and there don't appear to be any literal tab characters anywhere in the files.

Anyway, it looks better indented, so let's just bite the bullet once. It would be great if we both used the same formatter, then we could add, say, the `.prettierrc` file to source control to ensure we both format things with the same rules. I have vscode configured to run prettier on save.

File: user-overrides/All-variables.css
Comment (wrex) [reply user-overrides/All-variables.css:12]: I removed the UserStyle headers from the user override files (and also provided the link to the correct forum post!). Just publish the main stylesheet and user script stylesheets to `userstyles.world` when you are happy. Don't publish the user overrides -- we will just provide instructions for how users can create their own.

File: WKElementaryDark.css
Comment (Sepitus-exe) [reply WKElementaryDark.css:11]: It's not an issue. I was just lamenting about inconsistency when it comes to stylus's editor across platforms. On windows, there are no indents, linux has them and I can only imagine so does macOS.

Use any formatter you would like and I will try to match it. I have been doing so since the univers and everything update anyhow 😄

File: user-overrides/All-variables.css
Comment (Sepitus-exe) [reply user-overrides/All-variables.css]: I would actually like to have the overrides published and see them updated by their creators over time. Its just the template, that seemed dangerous, since its ment to be edited.

On that note, I would love for you to publish your own override yourself, to give you a total control over how, when and what happens with it. 😄

**Review by Sepitus-exe — APPROVED** [review]
Immaculate

**Pull request merged**
